crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -599,6 +599,7 @@ en:
           will be asked to upload it later on in the process.
           After you downloaded and saved the upgrade data click 'Continue Upgrade Process'.
         blockui_message: 'Aborting Upgrade...'
+        link: 'Manual download link:'
```

Comment:
Why is it "manual"? Or do you mean "Direct download link"?

(tbh, I'm not totally sure how it looks like, maybe a screenshot would help :-))